### PR TITLE
(DSP-1178) Add Impressum, edit landing page

### PR DESCRIPTION
**Pull request by SepidehAlassi**
resolves [DSP-1178](https://dasch.myjetbrains.com/youtrack/agiles/110-4/current?issue=DSP-1178)

- adds Impressum to the landing page
- edit the content of the landing page (e.g remove beta release stuff)
- edit contact info

### CODE REVIEW

**Review by gautschr — APPROVED** [review]
Looks good - thanks!

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -13,24 +13,26 @@
         <section class="text">
             <h1 class="mat-headline">Contact for Editorial Questions</h1>
 
-            Bernoulli Euler Center, University of Basel
-            <br/> Martin Mattmüller
-            <br/>
-            <a href="mailto:martin.mattmueller@unibas.ch">martin.mattmueller@unibas.ch</a>
-            <br/>
-
+            Bernoulli Euler Center (<a href="https://bez.unibas.ch/en/">BEZ</a>), University of Basel
```
Comment (flavens) [inline src/app/contact/contact.component.html]: I would open this webiste on a new tab, using target="_blank"

File: src/app/contact/contact.component.html [hunk 2]
```diff
@@ -13,24 +13,26 @@
...
         </section>
 
         <mat-divider></mat-divider>
 
         <section class="text">
-            <h2 class="mat-headline">Technical Contacts</h2>
+            <h2 class="mat-headline">Contact for Technical Questions</h2>
 
-            Digital Humanities Lab, University of Basel
+            Data and Service Center for the Humanities (<a href="https://dasch.swiss/">DaSCH</a>)
```
Comment (flavens) [inline src/app/contact/contact.component.html]: same here: I would open this website on a new tab, using target="_blank"

**Review by flavens — COMMENTED** [review]
it looks good, just. a few comments.
N.B. In the contact page, github should be written "GitHub"

**Comment by SepidehAlassi** [comment]
@flavens and @gautschr Thanks to both of you for your quick reviews.

**Pull request merged**